### PR TITLE
Add instrux to readme

### PR DESCRIPTION
**Pull request by izgeri**
### What does this PR do?
At current, this repo has an almost blank README that doesn't give much context to what it contains or how we'd recommend that people use it.

In addition, the main flow that will funnel people to this repo comes from the website, and we'd like an easy way to communicate to the web team when changes should be made to the website content.

In this PR, we add a duplicate of the website instructions to the repo README. This will allow:
- Visitors who land on this page from GitHub to run through the quick start tutorial from this repo if they'd like
- Visitors who land on this page from the website to be able to see in the README what the instructions are, if they prefer that format
- Developers to communicate to the web team when changes should be made to the web tutorial, by sharing a diff of this README content.

In addition, it adds the "Community" [certification level](https://http://github.com/cyberark/community/tree/master/Conjur/conventions/certification-levels.md) badge to this repo; users who want a Trusted method of deploying Conjur OSS would be advised to try out our [helm chart](https://github.com/cyberark/conjur-oss-helm-chart/).

### What ticket does this PR close?
n/a

#### Change log
- [ ] The CHANGELOG has been updated, or
- [X] This PR does not include user-facing changes and doesn't require a CHANGELOG update

#### Test coverage
- [ ] This PR includes new unit and integration tests to go with the code changes, or
- [X] The changes in this PR do not require tests

#### Documentation
- [X] Docs (e.g. `README`s) were updated in this PR, and/or there is a follow-on issue to update docs, or
- [ ] This PR does not require updating any documentation

### CODE REVIEW

File: .gitleaks.toml [hunk 1]
```diff
@@ -0,0 +1,185 @@
+title = "Conjur Quick Start gitleaks config"
```
[the remark this answers is not in the document]
Comment (izgeri) [reply .gitleaks.toml:1]: I needed to add an exception for `conf/certificate/nginx.key`

File: README.md [hunk 2]
```diff
@@ -1,10 +1,324 @@
...
+   Creating openssl ... done
+   Creating conjur_server ... done
+   Creating nginx_proxy ... done
+   Creating conjur_client ... done
+   ```
+
+   **Verification**
+   Run the following command to see a list of running containers:
+   ```
+   docker ps -a
```
[the remark this answers is not in the document]
Comment (izgeri) [reply README.md:131]: I have a lot of edits I want to make to this - some of them are reflected in @JakeQuilty and @alexkalish's suggestions in #19. Can you add this suggestion there too? My next step after getting this merged will be to create a PR with a changeset that I send to the web team to implement, so I can lump in any suggestions you have with that batch.

If it's easier, I could also send you a PR link and you could add your edits on top of that - but I want to get all of the changes into a single batch that we send to the web team, using the contents of this current PR as a baseline for a diff.

**Comment by izgeri** [comment]
@sashaCher I've marked this as "ready for review" if you're willing to approve it. I want to merge the PR as-is (modulo any discussion around the gitleaks config) because it's currently a mirror of the web "get started" flow.

In a future PR, I'll propose updates to the current flow that we can coordinate to go live in this repo and on the web at the same time. I'd love to include your suggestion changes in that future PR. Thanks!!

File: .gitleaks.toml (hunk 1 above)
[the remark this answers is not in the document]
Comment (sashaCher) [reply .gitleaks.toml:1]: @izgeri I've failed to find the file in the repo and it also does not present in the PR.
As far as I know `nginx.key` and `nginx.cert` files are generated on `docker-compose up` and are gitignored.
Are you sure that the file should be gitleaks ignored?

[the remark this answers is not in the document]
Comment (sashaCher) [reply .gitleaks.toml:1]: Please see https://github.com/cyberark/conjur-quickstart/pull/9

**Comment by sashaCher** [comment]
@izgeri No problem to approve the PR. Just let's clarify the reason why `.gitleaks.toml` file is there. It looks not related to the PR for me.

File: .gitleaks.toml (hunk 1 above)
[the remark this answers is not in the document]
Comment (izgeri) [reply .gitleaks.toml:1]: I agree the file no longer exists, but it _did_ once exist in the git history, and gitleaks complains about that.

when I run it against the source code, this is the output I get:
```
{"line":"-----BEGIN PRIVATE KEY-----","offender":"-----BEGIN PRIVATE KEY-----","commit":"c0c00b20d193ce62b27229dbf2c341c6b325e735","repo":".","rule":"Asymmetric Private Key","commitMessage":"Merge pull request #9 from cyberark/fix-ubuntu-nginx\n\nFix usage of this repo","author":"Srdjan Grubor","email":"sgnn7@sgnn7.org","file":"conf/tls/nginx.key","date":"2020-02-10T11:06:25-06:00","tags":"key, AsymmetricPrivateKey"}
{"line":"-----BEGIN PRIVATE KEY-----","offender":"-----BEGIN PRIVATE KEY-----","commit":"b94c1379ee98f43833e54ebed1cc059dbd669c2c","repo":".","rule":"Asymmetric Private Key","commitMessage":"Add template file for CONJUR_DATA_KEY\n","author":"orenbm","email":"oren.benmeir@cyberark.com","file":"conf/certificate/nginx.key","date":"2020-02-11T14:47:59+02:00","tags":"key, AsymmetricPrivateKey"}
{"line":"-----BEGIN PRIVATE KEY-----","offender":"-----BEGIN PRIVATE KEY-----","commit":"d829e35ba296472862b8d9013d99f5421e11e1ac","repo":".","rule":"Asymmetric Private Key","commitMessage":"Default deployment to work with 3rd party certificate\n","author":"Dvir Levanon","email":"dvirl@cyberark.com","file":"conf/tls/nginx.key","date":"2019-10-23T16:07:11+03:00","tags":"key, AsymmetricPrivateKey"}
{"line":"-----BEGIN PRIVATE KEY-----","offender":"-----BEGIN PRIVATE KEY-----","commit":"0298202d4eea1b8c7a922e47720fe78ff6a18256","repo":".","rule":"Asymmetric Private Key","commitMessage":"Modified gitignore file\n","author":"Yvonne Serota","email":"yvonne.serota@cyberark.com","file":"conf/tls/nginx.key","date":"2019-05-06T15:59:58+03:00","tags":"key, AsymmetricPrivateKey"}
{"line":"-----BEGIN PRIVATE KEY-----","offender":"-----BEGIN PRIVATE KEY-----","commit":"0d6a43304fb45df1572e70f7dbf3ad104162296c","repo":".","rule":"Asymmetric Private Key","commitMessage":"Made nginx certificate not be part of the repo\n\nSince they are dynamically generated, we don't want/need them in the\nrepo.\n","author":"Srdjan Grubor","email":"sgnn7@sgnn7.org","file":"conf/tls/nginx.key","date":"2020-01-31T20:17:41Z","tags":"key, AsymmetricPrivateKey"}
```

so there are 5 separate commits in the history all relating to this file (which I agree, has been removed). adding the gitleaks config with an explicit directive to ignore this now-non-existent file when it scans the history will allow me to continue running gitleaks on this project and push code up as needed.

**Comment by izgeri** [comment]
@sashaCher I updated the gitleaks config to specifically reference the 5 older commits that were problematic, which I think is a better approach.

**Pull request merged**